src/App.css

```diff
@@ -1,38 +1,156 @@
+
+.clearfix:after {visibility: hidden; display: block; font-size: 0; content: ''; clear: both; height: 0; }
+
+html { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+@font-face {
+  font-family: "Hind";
+ 
+  src: local("Hind-Regular"),
+    url("./fonts/Hind-Regular.ttf") format("truetype");
+}
+@font-face {
+  font-family: "Montserrat";
+
+  src: local("Montserrat"),
+    url("./fonts/Montserrat-VariableFont_wght.ttf") format("truetype");
+}
+
+
+html{
+  font-size: 62.5%;
+}
+
+body {
+  background-color: slateblue;
+  font-size: 2rem;
+}
+
+header {
+  text-align: center;
+}
+
+h1{
+  font-size: 6rem;
+  color: white;
+  font-family: Montserrat, sans-serif;
+}
+
+h2{
+  margin: 5px 1px 50px 1px;
+  font-family: Hind, sans-serif;
+  font-weight: 600;
+  font-size: 1.8rem;
+  color: darkslateblue;
+  text-transform: uppercase;
+}
+
+p {
+  font-size: 2rem;
+  font-family: Hind, sans-serif;
   text-align: center;
+  color: white;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+a {
+  text-decoration: none;
+  font-family: Hind, sans-serif;
+  font-size: 1.5rem;
+  font-weight: 600;
+  border: 2px solid darkslateblue;
+  padding: 2.5px 5px;
+  color: darkslateblue;
+  letter-spacing: -0.5px;
+  
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+a:hover{
```

Comment:
whenever we do a :hover on a link or button - usually i will add a :focus as well for accessibility purposes so if someone is using a keyboard to access the link it can be accessed / activated

Reply:
Agreed!